plugins/module_utils/turbo/server.py

```diff
@@ -307,14 +307,21 @@ def __init__(self):
         self.socket_path = None
         self.ttl = None
         self.debug_mode = None
+        self.jobs_started = 0
+        self.jobs_done = 0
```

Comment:
I feel like you could simplify this by just having a `self.task_running` boolean variable.

Reply:
We can have several tasks running in parallel.